### PR TITLE
feat(labels): allow thread row label title customization

### PR DESCRIPTION
**Pull request by dmy7r0**
## Description

Allows passing an HTML along with text for thread row labels.

### CODE REVIEW

File: src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js [hunk 1]
```diff
@@ -84,7 +84,11 @@ class GmailLabelView {
       return;
     }
 
-    querySelector(this._element, '.av').style.maxWidth = maxWidth;
+    try {
+      querySelector(this._element, '.av').style.maxWidth = maxWidth;
+    } catch (e) {
+      // noop
+    }
```
Comment (Macil) [inline src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js:91]:
```suggestion
    this._element.querySelector('.av')?.style.maxWidth = maxWidth;
```

The querySelector function imported from querySelectorOrFail is specifically a function built to throw on failure as opposed to the standard querySelector method, so this code would be more natural to do with the standard querySelector method if we don't want that behavior.

Comment (Macil) [reply src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js:91]: Wait, this is still a js file using Flow, and I don't think that supports optional chaining, unlike our Typescript files.

**Review by Macil — APPROVED** [review]

File: src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js (hunk 1 above)
Comment (wegry) [reply src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js:91]: This file shows off how hard it is to try converting Flow to TS piecemeal over years too. It's depended on by `gmail-thread-row-view` which has 5 flow files depending on it.

Comment (wegry) [reply src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js:91]:
```suggestion
    const childEl = this._element.querySelector('.av')
    if (childEl) {
      childEl.style.maxWidth = maxWidth;
    }
```

Comment (dmy7r0) [reply src/platform-implementation-js/dom-driver/gmail/widgets/gmail-label-view.js:91]: This looks great, thanks for the suggestions.

**Review by wegry — APPROVED** [review]

**Pull request merged**